aten/src/ATen/native/native_functions.yaml

```diff
@@ -3506,6 +3506,28 @@
     CPU, CUDA: sub_
     SparseCPU, SparseCUDA: sub_sparse_
 
+- func: heaviside.Tensor_out(Tensor self, Tensor val, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
```

Comment:
Do you need to make the dispatch explicit or will these kernels be found automatically?

Reply:
I'd make the kernel's naming style consistent with other methods.